### DISCUSSION TITLE
[Bug]  skywalking-skywalking-helm-oap  not started(keep restarting)

### DISCUSSION THREAD

**Discussion by wangchao732** [apache/skywalking#12173] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

OAP server (apache/skywalking)

### What happened

 helm install "${SKYWALKING_RELEASE_NAME}"   oci://registry-1.docker.io/apache/skywalking-helm   --version "${SKYWALKING_RELEASE_VERSION}"   -n "${SKYWALKING_RELEASE_NAMESPACE}"   --set oap.image.tag=9.6.0   --set oap.storageType=elasticsearch   --set ui.image.tag=9.6.0

### What you expected to happen

![image](https://github.com/apache/skywalking/assets/19440083/375f672e-8693-4002-8a10-46934cf4cd0a)
The logs show that this will restart

### How to reproduce

null

### Are you willing to submit a pull request to fix on your own?

- [ ] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wangchao732** [comment]
[oap.log](https://github.com/apache/skywalking/files/15148481/oap.log)
[oap (2).log](https://github.com/apache/skywalking/files/15148482/oap.2.log)

**Comment by freshgeek** [comment]
i run this commond line : helm install "${SKYWALKING_RELEASE_NAME}" oci://registry-1.docker.io/apache/skywalking-helm --version "${SKYWALKING_RELEASE_VERSION}" -n "${SKYWALKING_RELEASE_NAMESPACE}" --set oap.image.tag=9.6.0 --set oap.storageType=elasticsearch --set ui.image.tag=9.6.0

```
Error: INSTALLATION FAILED: failed to authorize: failed to fetch anonymous token: Get "https://auth.docker.io/token?scope=repository%3Aapache%2Fskywalking-helm%3Apull&service=registry.docker.io": EOF
```
